source/controllers/include/controllers/impedance/Impedance.hpp

```diff
@@ -0,0 +1,133 @@
+/**
```

Comment:
I generally prefer attribution to remain in source control rather than the source itself, but if you want to keep it there I won't stop you

Reply:
Right yes make sense. I never know if those are necessary and where they should go.